WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailActivity.java

```diff
@@ -148,16 +157,106 @@ protected void onSaveInstanceState(Bundle outState) {
         if (getSupportActionBar() != null && getSupportActionBar().getTitle() != null) {
             outState.putString(ARG_TITLE, getSupportActionBar().getTitle().toString());
         }
-
+        outState.putBoolean(NotificationsListFragment.NOTE_ALLOW_NAVIGATE_LIST_EXTRA, mAllowHorizontalNavigation);
+        outState.putString(NotificationsListFragment.NOTE_ID_EXTRA, mNoteId);
         super.onSaveInstanceState(outState);
     }
 
+    @Override
+    protected void onStart() {
+        super.onStart();
+        //if the user hasn't used swipe yet, hint the user they can navigate through notifications detail
+        //using swipe on the ViewPager
+        if (!AppPrefs.isSwipeToNavigateShown()) {
```

Comment:
Are there situations where only one note is shown, so there's no swiping? If so, we should make sure not to show this snackbar in those situations.